src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/altar/AncientPedestal.java

```diff
@@ -62,17 +73,21 @@ public AncientPedestal(ItemGroup itemGroup, SlimefunItemStack item, RecipeType r
 
             @Override
             public void onBlockBreak(@Nonnull Block b) {
+
+
                 Optional<Item> entity = getPlacedItem(b);
 
                 if (entity.isPresent()) {
                     Item stack = entity.get();
 
                     if (stack.isValid()) {
+                        getArmorStand(b).removePassenger(stack);
```

Comment:
Not sure this is needed? You're killing the item entity which should be enough?

Reply:
Yup was not needed
resolved in latest commit